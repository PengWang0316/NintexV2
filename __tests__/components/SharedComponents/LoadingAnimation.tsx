import React from 'react';
import renderer from 'react-test-renderer';

import { LoadingAnimation } from '../../../app/components/SharedComponents/LoadingAnimation';
import JssProviderWrapper from '../../libs/JssProviderWrapper';

describe('LoadingAnimation component', () => {
  const defaultProps = { isLoading: true };
  test('Snapshot with isLoading true', () => expect(renderer.create(JssProviderWrapper(<LoadingAnimation {...{ ...defaultProps }} />)).toJSON()).toMatchSnapshot());
  test('Snapshot with isLoading false', () => expect(renderer.create(JssProviderWrapper(<LoadingAnimation {...{ ...defaultProps, isLoading: false }} />)).toJSON()).toMatchSnapshot());
});
