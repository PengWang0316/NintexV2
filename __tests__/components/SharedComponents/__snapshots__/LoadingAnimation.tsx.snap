// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingAnimation component Snapshot with isLoading false 1`] = `null`;

exports[`LoadingAnimation component Snapshot with isLoading true 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="makeStyles-div-2"
  >
    <div
      className="MuiLinearProgress-root MuiLinearProgress-colorPrimary MuiLinearProgress-indeterminate"
      role="progressbar"
    >
      <div
        className="MuiLinearProgress-bar MuiLinearProgress-bar1Indeterminate MuiLinearProgress-barColorPrimary"
        style={Object {}}
      />
      <div
        className="MuiLinearProgress-bar MuiLinearProgress-bar2Indeterminate MuiLinearProgress-barColorPrimary"
        style={Object {}}
      />
    </div>
  </div>
</div>
`;
